apps/dashboard/src/lib/components/Courses/index.svelte

```diff
@@ -32,6 +32,7 @@
     {#each courses as courseData}
       {#key courseData.id}
         <Card
+          showContextMenu
```

Comment:
Thank you for adding this. You really figured it the right way to implement this so students don't get this option.

Reply:
Thanks @rotimi-best